### DISCUSSION TITLE
Problem With Github Student Package

### DISCUSSION THREAD

**Discussion by HalemoGPA** [community/community#120979] [GitHub Education]
### Github Student Package

Bug

### Body

When I try to sign in student package, it asks for ID card from university. When I capture the front face of the card it works well and identifies me. The problem happens when I capture the back face of the card. It can't recognize it :(

**Comment by haululazkiyaa** [comment]
I have same problem with you, but i have solution for you that you..
You can try use a desktop/laptop and use their webcam or use external webcam. 
Don't use a back smartphone camera, because its buggy.
